packages/flutter_tools/templates/module/android/library/Flutter.tmpl/src/main/AndroidManifest.xml.tmpl

```diff
@@ -1,5 +1,11 @@
 <!-- Generated file. Do not edit. -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="{{androidIdentifier}}">
+    package="{{androidIdentifier}}"
+    xmlns:tools="http://schemas.android.com/tools">
   <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
uber nit: feel free to do it in a different PR / not do it if you feel it's not necessary: while you're here, we can probably split this into a debug and release manifest. 

Only debug needs the internet permission (for the observatory) and only release needs this metadata.

Reply:
sg. feel free to create an issue for that :)